frontend/src/features/permits/pages/Application/components/review/PermitReview.tsx

```diff
@@ -128,6 +128,8 @@ export const PermitReview = (props: PermitReviewProps) => {
           }
           handleApproveButton={props.handleApproveButton}
           handleRejectButton={props.handleRejectButton}
+          startDate={props.permitStartDate}
```

Comment:
Instead of passing the startDate and expiryDate in to the ReviewActions component, refactor the ReviewActions props so that it accepts "disableApproveButton" AND "disableRejectButton" as props, and pass in the result of the check to these props instead